src/main/kotlin/uk/gov/dluhc/printapi/rds/mapper/InstantMapper.kt

```diff
@@ -0,0 +1,10 @@
+package uk.gov.dluhc.printapi.rds.mapper
+
+import org.springframework.stereotype.Component
+import java.time.Instant
+import java.time.OffsetDateTime
+
+@Component
+class InstantMapper {
+    fun toInstant(offsetDateTime: OffsetDateTime?): Instant? = offsetDateTime?.toInstant()
+}
```

Comment:
Do these mapper belong to `rds` package?

Reply:
oh may be these are all rds (mysql) related changes 👍🏼 . Ignore above comment then

Reply:
Just in `rds` package to keep it clear at this stage which are used for RDS... a future task will move to correct package 👍